Targets/njs/README.md

```diff
@@ -0,0 +1,11 @@
+# Target: NJS
+
+To build njs for fuzzing:
+* Step 1 - prepare env
+    * Run `fuzzbuild.sh`, this will:
```

Comment:
So typically, `fuzzbuild.sh` just takes care of performing the build, not downloading the source code. You could rename this to maybe `setup.sh` or so, and then put steps 2 and 3 into a `fuzzbuild.sh`. That would be more consistent. No strong opinion though, this is also fine.